packages/hedera-identify-snap/packages/site/gatsby-ssr.tsx

```diff
@@ -0,0 +1,34 @@
+/*-
+ *
+ * Hedera Wallet Snap
```

Comment:
Update all license headers to be correct for this package
```suggestion
 * Hedera Identify Snap
```